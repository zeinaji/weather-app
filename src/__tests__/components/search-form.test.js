import React from "react";
import { render } from "@testing-library/react";
import SearchForm from "../../components/search-form";

describe("SearchForm", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<SearchForm />);
    expect(asFragment()).toMatchSnapshot();
  });
});
