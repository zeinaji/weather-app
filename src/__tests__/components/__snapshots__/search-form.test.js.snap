// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm renders correctly 1`] = `
<DocumentFragment>
  <div
    class="search-form"
    style="margin: 25px 25px 0px 0px;"
  >
    <input
      style="border-radius: 5px; border: 0.5px solid rgb(53, 51, 51);"
      type="text"
      value=""
    />
    <button
      class="search-button"
      style="font-family: Balsamiq Sans, cursive; border-radius: 5px; border: 0.5px solid rgb(53, 51, 51); margin-left: 3px; cursor: pointer; color: rgb(53, 51, 51);"
    >
      Search
    </button>
  </div>
</DocumentFragment>
`;
